docker-compose.yml

```diff
@@ -10,13 +10,19 @@ services:
     tty: true
     volumes:
       - ./.env:/home/node/app/.env
+    environment:
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+
   relay-ws:
     container_name: hedera-json-rpc-relay-ws
     image: "ghcr.io/hashgraph/hedera-json-rpc-relay:main"
     command: start:ws
     environment:
       HEALTHCHECK_PORT: 8547
       SUBSCRIPTIONS_ENABLED: true
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
```

Comment:
don't think that the `relay-ws` has those configurations either.

https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/docs/configuration.md#ws-server

Reply:
the `relay-ws` is using an instance of the `relay` package under the hood, so it will use the same configuration from the relay.

Reply:
done, please check